lib/bunny/session.rb

```diff
@@ -1257,7 +1257,7 @@ def heartbeat_disabled?(val)
 
     # @private
     def negotiate_value(client_value, server_value)
-      return server_value if client_value == :server
+      return server_value if [:server, "server"].include? client_value
```

Comment:
Sorry to nitpick but can you please use brackets around the argument?

Reply:
sure I will do so.